delivery/common/cwt/examples/typescript/src/cwt/cwt.js

```diff
@@ -0,0 +1,194 @@
+/** @preserve @version 1.1.0 */
+import { crypto } from "crypto";
+
+import { Encoder, Decoder } from "./cbor-x.js";
+
+const COSE_Mac0 = 17, COSE_Mac = 97, COSE_Sign = 98, COSE_Sign1 = 18, COSE_Encrypt0 = 16, COSE_Encrypt = 96, coseAlgTags = {
+    5: "HMAC 256/256",
+    "-7": "ES256"
+}, HeaderLabelToKey_alg = 1, HeaderLabelToKey_crit = 2, claimsLabelToKey_iss = 1, claimsLabelToKey_sub = 2, claimsLabelToKey_aud = 3, claimsLabelToKey_exp = 4, claimsLabelToKey_nbf = 5;
+
+class Mac {
+    static async verifyHMAC(alg, message, signature, keys) {
+        if ("HMAC 256/256" === alg) {
+            let isSignVerified = !1;
+            for (const key of keys) if (isSignVerified = await crypto.subtle.verify({
+                name: "HMAC"
+            }, key, signature, message), isSignVerified) return Promise.resolve(isSignVerified);
+            return Promise.resolve(isSignVerified);
```

Comment:
The function is async, so I don't think it's necessary to create a resolved Promise.